tools/lib/template_parser.py

```diff
@@ -138,8 +138,8 @@ def looking_at_whitespace() -> bool:
                 kind = "django_comment"
             elif looking_at_handlebarpartial():
                 s = get_handlebar_partial(text, state.i)
-                tag = s[9:-2]
-                kind = "handlebars_singleton"
+                tag = s[5:-2]
+                kind = "handlebars_partial"
```

Comment:
If it's easy to add a test in `tools/tests/test_template_parser.py`, that's an important part of how I'd make changes to this library, since @showell is our expert on it.

Also, are you sure this doesn't break anything else? We do use the `{{>` syntax elsewhere, and I'm kinda guessing that you should be adding a new block, not changing that existing one.

Reply:
It's confusing to me that the old one also called it a partial, but I decided not to look into that for now and just rename the old one to `singleton` more consistently. Also added tests.

Reply:
How did you pick these names? It looks like the thing we already have is a "partial" and the thing you're adding is a "partial block", so maybe we should use those names?

https://handlebarsjs.com/guide/partials.html#partial-blocks

Reply:
I think the original `handlebars_singleton` name is the thing that's confusing -- can you rename it to be consistently `handlebars_partial` and the new one to `handlebars_partial_block`? Then we're using Handlebars upstream terminology for these two concepts, rather than whatever was made up when working on this tool.